modules/deploy/pages/deployment-option/self-hosted/kubernetes/kubernetes-cluster-requirements.adoc

```diff
@@ -103,109 +106,11 @@ Redpanda supports the following storage providers for Tiered Storage:
-| Schema registry
-
-| 30082
-| HTTP Proxy
-
-| 31092
-| Kafka API
-
-| 31644
-| Admin API
-|===
-
-To change these ports, see xref:manage:kubernetes/networking/configure-listeners.adoc[].
-
-Depending on your deployment and security policies, you may not be able to access worker nodes through a NodePort Service.
-If you choose to use another Service, consider the impact on the cost and performance of your deployment:
-
-* *LoadBalancer Service*: To make each Redpanda broker accessible with LoadBalancer Services, you need one LoadBalancer Service for each Redpanda broker so that requests can be routed to specific brokers rather than balancing requests across all brokers. Load balancers are expensive, add latency and occasional packet loss, and add an unnecessary layer of complexity.
-* *Ingress*: To make each Redpanda broker accessible with Ingress, you must run an Ingress controller and set up routing to each Redpanda broker. Routing adds latency and can be a throughput bottleneck.
-
-For more details, see xref:manage:kubernetes/networking/networking-and-connectivity.adoc[Networking and Connectivity].
-
-=== Use ExternalDNS for external access
-
-Redpanda Data recommends using ExternalDNS to manage DNS records for your Pods' domains. ExternalDNS synchronizes exposed Kubernetes Services with various DNS providers, rendering Kubernetes resources accessible through DNS servers.
-
-Benefits of ExternalDNS include:
-
-* *Automation*: ExternalDNS automatically configures public DNS records when you create, update, or delete Kubernetes Services or Ingresses. This eliminates the need for manual DNS configuration, which can be error-prone.
-* *Compatibility*: ExternalDNS is compatible with a wide range of DNS providers, including major cloud providers such as AWS, Google Cloud, and Azure, and DNS servers like CoreDNS and PowerDNS.
-* *Integration with other tools*: ExternalDNS can be used in conjunction with other Kubernetes tools, such as ingress controllers or cert-manager for managing TLS certificates.
-
-You can use ExternalDNS with the default xref:manage:kubernetes/networking/configure-external-access-nodeport.adoc#externaldns[NodePort Service] or with xref:manage:kubernetes/networking/configure-external-access-loadbalancer.adoc#externaldns[LoadBalancer Services].
-
-=== Secure your cluster
-
-Deploy Redpanda in a separate namespace to protect your data from other resources in your Kubernetes cluster.
-
-To protect your Redpanda cluster, enable and configure the following:
-
-* xref:manage:kubernetes/security/sasl-kubernetes.adoc[Authentication through SASL]
-* xref:manage:kubernetes/security/kubernetes-tls.adoc[Network encryption through TLS]
+include::shared:partial$suggested-reading.adoc[]
 
-By default, the Redpanda Helm chart enables TLS using cert-manager.
+- xref:./redpanda-cluster-recommendations.adoc[]
+- xref:reference:redpanda-helm-spec.adoc#rackawareness[Redpanda Helm Specification]
+- xref:reference:crd.adoc[Redpanda CRD Reference]
```

Comment:
suggest removing the final period from the Next steps
[Deploy Redpanda in Kubernetes](https://deploy-preview-36--redpanda-docs-preview.netlify.app/current/deploy/deployment-option/self-hosted/kubernetes/kubernetes-deploy/).
As a guideline, I think only inline links should have closing periods, not links listed in Suggested Reading or Next Steps